mycity/mycity/intents/food_truck_intent.py

```diff
@@ -0,0 +1,190 @@
+                       'Hours': '11 a.m. - 3 p.m.',
+                       'Link': 'http://www.morockinfusion.com/',
+                       'Loc': 'Chinatown Park',
+                       'Management': 'Rose Kennedy Greenway Conservancy',
+                       'Notes': ' ',
+                       'POINT_X': -7910324.35486,
+                       'POINT_Y': 5213745.7545,
+                       'Site_num': 0,
+                       'Start_time': '11:00:00 AM',
+                       'Time': 'Lunch',
+                       'Title': 'Greenway',
+                       'Truck': "Mo'Rockin Fusion"},
+       'geometry': {'x': -71.05965270349564, 'y': 42.351281064296}
+    }
+    """
+    base_url = 'https://services.arcgis.com/sFnw0xNflSi8J0uh/arcgis/rest/' \
+               'services/food_trucks_schedule/FeatureServer/0/query'
+    day_of_week = "Day='" + datetime.datetime.today().strftime('%A') + "' "
+    url_params = {
+        "f": "json",
+        "outFields": "*",
+        "outSR": "4326",
+        "returnGeometry": "true",
+        "where": day_of_week
+    }
+    logger.debug("Requesting data from ArcGIS server!")
+    food_truck_data_result = requests.get(base_url, url_params)
+    response_code = food_truck_data_result.status_code
+    logger.debug("Got response code: " + str(response_code))
+    if response_code != requests.codes.ok:
+        logger.debug('HTTP Error: '.format(response_code))
+        return {}
+
+    print(food_truck_data_result)
+
+    # Generate unique list of food truck locations
+    # to send to the Google Maps API
+    trucks = food_truck_data_result.json()['features']
+    truck_unique_locations = []
+    for truck in trucks:
+        if truck["attributes"]["Loc"] not in truck_unique_locations:
+            truck_unique_locations.append([truck["geometry"],
+                                           truck["attributes"]["Truck"],
+                                           truck["attributes"]["Loc"],
+                                           truck["attributes"]["Title"],
+                                           truck["attributes"]["Start_time"],
+                                           truck["attributes"]["End_time"]])
+    return truck_unique_locations
+
+
+def get_nearby_food_trucks(mycity_request):
+    """
+    Gets food truck info near an address
+
+    :param mycity_request: MyCityRequestDataModel object
+    :return: MyCityResponseObject
+    """
+    mycity_response = MyCityResponseDataModel()
+
+    # Get current address location
+    if CURRENT_ADDRESS_KEY in mycity_request.session_attributes:
+        current_address = \
+            mycity_request.session_attributes[CURRENT_ADDRESS_KEY]
+
+        address_parser = StreetAddressParser()
+        a = address_parser.parse(current_address)
+        address = str(a['house']) + " " + str(a['street_name']) + " " \
+                  + str(a['street_type'])
+        zip_code = str(a["other"]).zfill(5) if a["other"] else None
```

Comment:
I forget whether we were favoring single quotes or double but choose one or the other here